### DISCUSSION TITLE
CS5001:	Program does not contain a static 'Main' method suitable for an entry point

### DISCUSSION THREAD

**Discussion by iomismo** [microsoft/WindowsAppSDK#1873] [Q&A]
Hi everyone!
Weird error: any Windows App SDK project that is created from VS 2022 / 2019 Templates throws CS5001 error. When creating a new project it first shows app.manifest, App.xaml and MainWindow.xaml files and after a second they are automatically removed from the project tree. Adding them mannually does not solve the problem.

Thought that it was only on my old Project Reunion projects after upgrading them to Windows App SDK 1.0 and .NET 6, but it happens even creating a new project / solution, no matter VS 2019 or 2022. 

Code inside App.g.cs or any other page g.cs file is not being generated.
In App.i.g.cs file, the autogenerated Program class that contains Main method is missing. 

Cleaned, rebuild, deleted obj and bin folders, not working.

**Comment by btueffers** [comment]
Hey @iomismo! We've created an [issue](https://github.com/microsoft/WindowsAppSDK/issues/1877) from this discussion and assigned a few folks. We'll share further updates there!
